examples/deployment/src/asa-helpers.js

```diff
@@ -58,6 +58,34 @@ exports.transferMicroAlgos = async function (deployer, fromAccount, toAccountAdd
   return await deployer.waitForConfirmation(pendingTx.txId)
 }
 
+// Transfer ALGO
+exports.transferMicroAlgosContract = async function (deployer, fromAccount, toAccountAddr, amountMicroAlgos, lsig) {
+
+  let params = await deployer.algodClient.getTransactionParams().do();
+
+  params.fee = 0;
+  params.flatFee = true;
```

Comment:
you don't need to specify it. Default values should be OK.

Reply:
ok